R/shiny.R

```diff
@@ -373,7 +373,7 @@ rmarkdown_shiny_ui <- function(dir, file) {
           id = "rmd_loader_wrapper",
           tags$div(id = "rmd_loader", style = "display: none",
                    tags$img(src = "rmd_resources/rmd_loader.gif"),
-                   tags$p("Loading")))),
+                   tags$p("Please wait...")))),
```

Comment:
Please move the text above the gif (that is what ShinyApps.io shows on the loading page).